CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 
```

Comment:
not sure if changelog should also comment on breaking changes?

Reply:
copied some ethermint changelog.